app/app.go

```diff
@@ -26,9 +26,8 @@ import (
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/vesting"
 	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
-	"github.com/cosmos/cosmos-sdk/x/bank"
-	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
-	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	sdkbank "github.com/cosmos/cosmos-sdk/x/bank"
```

Comment:
why rename this?

Reply:
There is another import for wrapped bank created by alliance.
https://github.com/Stride-Labs/stride/pull/582/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R55

Reply:
1. Should alliance's bank replace all instances of bank in our app.go? (I see a single line in which we still use sdkbank, line 183)
2. Might be most clear to import alliance's as `alliancebank` or similar, rather than as `bank`

Reply:
ok